crates/trigger/src/cli.rs

```diff
@@ -96,6 +97,9 @@ where
     /// Load the application from the registry.
     #[clap(long = "from-registry", hide = true)]
     pub from_registry: bool,
+    // /// The root directory of the registry cache.
+    // #[clap(long = "registry-cache-root", hide = true)]
+    // pub registry_cache_root: Option<PathBuf>,
```

Comment:
I was lazy here...It was easier to pass this from `spin up` using an environment variable....